lib/tapjoy/autoscaling_bootstrap/configure_autoscaler.rb

```diff
@@ -16,6 +20,11 @@ def initialize(misc_config:, aws_env:, user_data:, **unused_values)
 
       private
 
+      # Check if ASG exists so we know whether to create or update
+      def autoscaling_group_exists
+        !Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil?
```

Comment:
My preference is for `Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil? ? false : true` Stylistically you should consider changing the method name to `autoscaling_group_exists?` which indicates a boolean will be returned.